domains/nura.is-not-a.dev.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Email Configuration",
+    "domain": "is-not-a.dev",
+    "subdomain": "nura",
+    "record": {
+        "MX": ["aspmx1.migadu.com"],
+        "MX": ["aspmx2.migadu.com"],
+        "TXT": ["hosted-email-verify=fvlpvxpb"],
+        "TXT": ["v=spf1 include:spf.migadu.com -all"]
```

Comment:
Only one property each. Arrays can have multiple strings. See DNS rules for more information.